securedrop/journalist.py

```diff
@@ -698,16 +709,15 @@ def reply():
            collection view, regardless if the Reply is created
            successfully.
     """
-    msg = request.form['msg']
-    # Reject empty replies
-    if not msg:
+    form = ReplyForm()
+    if not form.validate_on_submit():
         flash(gettext("You cannot send an empty reply."), "error")
```

Comment:
This call to **flash** displays a message that is redundant with [the above ReplyForm](https://github.com/freedomofpress/securedrop/pull/2427/files#diff-2af57f469c54eb2e19fbb1a0e2a5b35aR54), could you please make it so there is no redundancy ? Thanks :-)